observations/management/commands/import_swimming_water_temperatures.py

```
@@ -0,0 +1,289 @@
+            sorted(known_unit_ids), base_url, observable_property, dry_run
+        )
+        self._report(created, units_updated, errors)
+
+    def _get_property(self):
+        try:
+            return ObservableProperty.objects.get(pk=PROPERTY_ID)
+        except ObservableProperty.DoesNotExist:
+            raise CommandError(
+                f"ObservableProperty '{PROPERTY_ID}' does not exist. "
+                "Run migrations or load the observation fixtures first."
+            )
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)